src/OrchardCore.Modules/OrchardCore.OpenId/Drivers/OpenIdServerSettingsDisplayDriver.cs

```diff
@@ -109,18 +114,65 @@ public override async Task<IDisplayResult> UpdateAsync(OpenIdServerSettings sett
                 settings.TestingModeEnabled = model.TestingModeEnabled;
                 settings.AccessTokenFormat = model.AccessTokenFormat;
                 settings.Authority = model.Authority;
+
                 settings.CertificateStoreLocation = model.CertificateStoreLocation;
                 settings.CertificateStoreName = model.CertificateStoreName;
                 settings.CertificateThumbprint = model.CertificateThumbprint;
-                settings.EnableTokenEndpoint = model.EnableTokenEndpoint;
-                settings.EnableAuthorizationEndpoint = model.EnableAuthorizationEndpoint;
-                settings.EnableLogoutEndpoint = model.EnableLogoutEndpoint;
-                settings.EnableUserInfoEndpoint = model.EnableUserInfoEndpoint;
-                settings.AllowPasswordFlow = model.AllowPasswordFlow;
-                settings.AllowClientCredentialsFlow = model.AllowClientCredentialsFlow;
-                settings.AllowAuthorizationCodeFlow = model.AllowAuthorizationCodeFlow;
-                settings.AllowRefreshTokenFlow = model.AllowRefreshTokenFlow;
-                settings.AllowImplicitFlow = model.AllowImplicitFlow;
+
+                settings.AuthorizationEndpointPath = model.EnableAuthorizationEndpoint ?
```

Comment:
The endpoint paths are now stored in `OpenIdServerSettings` but I didn't make them configurable via the UI. It's something we'll be able to do in the future without breaking anything.